### PR TITLE
Create guide to deploy to Railway

### PR DESCRIPTION
**Pull request by abuuzayr**
Added guide to deploy blitz to [Railway](https://railway.app)

### CODE REVIEW

**Comment by abuuzayr** [comment]
Ready for review @JuanM04 !

**Comment by flybayer** [comment]
Thanks!! My only suggestion is to make it a bit more clear that about the various sections. I.e. the first section is required if you don't already have a railway account. Then also make it clearer that you have two options for project setup: through the UI or through the CLI.

**Comment by abuuzayr** [comment]
> Thanks!! My only suggestion is to make it a bit more clear that about the various sections. I.e. the first section is required if you don't already have a railway account. Then also make it clearer that you have two options for project setup: through the UI or through the CLI.

Absolutely! Great points, thank you for the review. Will push the changes up.

File: app/pages/docs/deploy-railway.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+```
+npm i -g @railway/cli
+```
+
+or
+
+```
+yarn global add @railway/cli
+```
```
Comment (JuanM04) [inline app/pages/docs/deploy-railway.mdx:36]:
``````suggestion
```bash
npm i -g @railway/cli
# or
yarn global add @railway/cli
```
``````

**Pull request merged**